lib/issuer.js

```diff
@@ -274,7 +274,10 @@ function getCredentialSigningKey(credential, callback) {
             'Preferred credential signing key not found.', 'NotFound',
             {httpStatusCode: 404, public: true}, err));
         }
-        if(publicKey.sysStatus !== 'active') {
+        // privateKey may be undefined because it either does not exist
+        // or the issuer identity does not have the proper permissions to
+        // access the private key
+        if(!privateKey || publicKey.sysStatus !== 'active') {
```

Comment:
I see you put this under "invalid" instead of "not found"... and I think I agree.